project_timeline_critical_path/models/project_task.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Onestein
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
```

Comment:
Could you sort imports alphabetically? https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports